src/mixins/createProjectorMixin.ts

```diff
@@ -142,6 +143,10 @@ function attach(instance: Projector, { type }: AttachOptions) {
 
 	projectorData.attachPromise = new Promise((resolve, reject) => {
 		projectorData.afterCreate = () => {
+			if (projectorData.originalAfterCreate) {
+				projectorData.originalAfterCreate.call(instance);
```

Comment:
No you're right, I should pass those in just in case the original call needs them. Good catch!